homework/01LessonOne.js

```diff
@@ -16,28 +16,29 @@ var myUndefined;
 // Let's create a new variable named myNull and give it the value null.
 //
 // What is the difference between null and undefined?
+// Null is for JavaScript objects, i.e., for something to be null, an object has to exist. Null is basically a placeholder assignment value. Undefined is for JavaScript variables, and means that no value has been declared/defined to a variable.
```

Comment:
Very strong start, @kckimberlycook!

However, any variable in JavaScript can be **undefined** or **null**.  **null** is often used as a way to indicate that a variable or property has been initialized, but not given a value.